packages/bridge/src/hooks/withdraw/useWithdraw.tsx

```diff
@@ -138,14 +138,24 @@ export const useWithdraw = (params: UseWithdrawProps) => {
   }, [withdrawApi, stage, params, registerInteractiveFeedback]);
 
   const canCancel = useMemo(() => {
-    return stage < WithdrawState.SendingTransaction && stage > WithdrawState.Ideal;
+    const isBeforeSendingTX = stage < WithdrawState.SendingTransaction;
+    const actionStarted = stage > WithdrawState.Ideal;
+    const actionEnded = stage > WithdrawState.SendingTransaction;
+    const canCancel = isBeforeSendingTX && actionStarted;
+    return canCancel || actionEnded;
```

Comment:
This is for resting the state

Reply:
I don't think it makes sense to cancel anything in a resting state, can the actionEnded be removed?

Reply:
what this for is the `cancel` button,  
the user can cancel if the `canCancel` ,  
or `actionEnded` where the `cancel` button has another context `confirm` for success tx and close for failure

Reply:
please check this [https://github.com/webb-tools/webb-dapp/blob/4de12ecc046a76976e1328931a4f018473db86db/packages/bridge/src/hooks/withdraw/useWithdraw.tsx#L149](https://github.com/webb-tools/webb-dapp/blob/4de12ecc046a76976e1328931a4f018473db86db/packages/bridge/src/hooks/withdraw/useWithdraw.tsx#L149)  
this will clear it for you

Reply:
A task was created for this: #https://github.com/webb-tools/webb-dapp/issues/251